v1.1/array.md

```diff
@@ -0,0 +1,131 @@
+---
+title: ARRAYS
+summary: The ARRAYS data type stores 
+toc: false
+---
+
+<span class="version-tag">New in v1.1:</span>The `ARRAY` data type stores one-dimensional, 1-indexed, homogenous arrays of any non-array [data type](data-types.html).
```

Comment:
In response to @knz note that arrays aren't great in sql, we could add a second paragraph like:

> The `ARRAY` type is useful for ensuring compatibility with ORMs and other tools. However, if such compatibility is not a concern, it's more flexible to design your schema with normalized tables.

@knz, does that sound ok?

Reply:
Fixed.